CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Current Master
 
+- Create visible difference in transaction history between a approved but not yet included in a block transaction and a transaction who has been confirmed.
```

Comment:
Rewording:
`Create a visible difference in the transaction history between a transaction that is approved but not yet included in a block, and one that has been confirmed.`